src/test/java/net/minestom/server/instance/ExplosionIntegrationTest.java

```diff
@@ -0,0 +1,62 @@
+package net.minestom.server.instance;
+
+import net.kyori.adventure.nbt.CompoundBinaryTag;
+import net.minestom.server.coordinate.Point;
+import net.minestom.server.coordinate.Pos;
+import net.minestom.server.network.packet.server.play.ExplosionPacket;
+import net.minestom.testing.Env;
+import net.minestom.testing.extension.MicrotusExtension;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import java.util.List;
+
+
+@ExtendWith(MicrotusExtension.class)
+public class ExplosionIntegrationTest {
```

Comment:
In JUnit 5, it's not necessary to use the `public` keyword for class declarations or test methods. You can remove it from your test classes and methods.

In Chapter [2.3](https://junit.org/junit5/docs/current/user-guide/), you can find some information about that behaviour

Reply:
Thank you for the valuable feedback! I've pushed the changes.